app/src/main/java/io/intrepid/contest/screens/contestcreation/describecontest/DescribeContestFragment.java

```diff
@@ -0,0 +1,49 @@
+package io.intrepid.contest.screens.contestcreation.describecontest;
+
+import android.content.Context;
+import android.os.Bundle;
+import android.support.annotation.NonNull;
+import android.support.annotation.Nullable;
+import android.widget.EditText;
+
+import butterknife.BindView;
+import io.intrepid.contest.R;
+import io.intrepid.contest.base.BaseFragment;
+import io.intrepid.contest.base.PresenterConfiguration;
+import io.intrepid.contest.models.Contest;
+import io.intrepid.contest.screens.contestcreation.ContestActivityContract;
+import io.intrepid.contest.screens.contestcreation.ContestCreationFragment;
+import io.intrepid.contest.screens.contestcreation.ValidatableView;
+import io.intrepid.contest.screens.contestcreation.namecontest.NameContestFragment;
+
+public class DescribeContestFragment extends BaseFragment<DescribeContestPresenter> implements DescribeContestContract.View, ContestCreationFragment{
+    @BindView(R.id.contest_description_edittext)
+    EditText descriptionField;
+
+    @Override
+    protected int getLayoutResourceId() {
+        return R.layout.fragment_edit_contest_description;
+    }
+
+    @NonNull
+    @Override
+    public DescribeContestPresenter createPresenter(PresenterConfiguration configuration) {
+        return new DescribeContestPresenter(this, configuration);
+    }
+
+    @Override
+    protected void onViewCreated(@Nullable Bundle savedInstanceState) {
+        super.onViewCreated(savedInstanceState);
+    }
+
+    @Override
+    public void onNext() {
+        presenter.onContestDescribed(descriptionField.getText().toString());
```

Comment:
Lets change the name to presenter.onNextClicked, passing in the field is fine though.